pennylane_lightning/core/src/utils/cuda_utils/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+project(cuda_utils LANGUAGES CXX CUDA)
+set(CMAKE_CXX_STANDARD 20)
+
+findCUDATK(lightning_compile_options)
+
+add_library(cuda_utils INTERFACE)
```

Comment:
We may want to link this in `lightning_gpu_utils` and the future `lightning_other_cuda_backend_utils` devices so that we can remove `cuda_utils` from the root CMakeLists. what do you think?

Reply:
Hey @maliasadi , what about linking `cuda_utils` to `lightning_utils`. We can remove the explicit dependency on `cuda_utils` in the root CMakelists as well.